app/models/responsible_body.rb

```diff
@@ -232,11 +231,11 @@ def next_school_sorted_ascending_by_name(school)
   end
 
   def orders_managed_centrally?
-    who_will_order_devices == 'responsible_body'
+    default_who_will_order_devices_for_schools == 'responsible_body'
   end
 
   def orders_managed_by_schools?
```

Comment:
It seems that the value `schools` has been cleaned up in production and we should be able to get rid of this.

```
[25] pry(main)> School.where(who_will_order_devices: 'school').count
=> 17191
[26] pry(main)> School.where(who_will_order_devices: 'schools').count
=> 0
```

Reply:
```
[29] pry(main)> ResponsibleBody.where(who_will_order_devices: 'schools').count
=> 0
[30] pry(main)> ResponsibleBody.where(who_will_order_devices: 'school').count
=> 3043
```